apps/condo/domains/organization/components/OrganizationSelect.tsx

```diff
@@ -121,7 +121,7 @@ export const OrganizationSelect: React.FC = () => {
                         ref={selectRef}
                         css={blackSelectCss}
                         size={'middle'}
-                        showAction={['focus', 'click' ]}
+                        showAction={['focus', 'click']}
```

Comment:
I don't shure but in new codestyle we should to move lines like this to constant expression